src/cosalib/ova.py

```diff
@@ -90,6 +90,10 @@ def generate_ovf_parameters(self, vmdk, cpu=2, memory=4096):
         with open(os.path.join(self._workdir, 'tmp/image.json')) as f:
             image_json = json.load(f)
 
+        secure_boot = "true"
```

Comment:
The default should go in `image-default.yaml`, and then we won't need to hardcode it here.

Reply:
I set it for the default in FCOS here https://github.com/coreos/fedora-coreos-config/pull/1837 to apply to all streams. For RHCOS I just set it in image.yaml https://github.com/openshift/os/pull/888